benchmark/realnest/07_select_text.benchmark

```diff
@@ -0,0 +1,18 @@
+# name: benchmark/realnest/07_select_text.benchmark
+# description: Creates a list of unnested texts, counts texts string's length and returns a number of unique values
```

Comment:
Creates a list by unnesting and re-aggregating it, then performs `list_transform` and `list_unique` on that list.